gef-ui/server/services/__tests__/api.test.js

```diff
@@ -236,7 +236,7 @@ describe('deleteFacility()', () => {
     await expect(api.deleteFacility({ facilityId: validMongoId, userToken })).rejects.toThrowError();
   });
 
-  test.each(invalidMongoIdTestCases)('returns false when given an invalid facilityId', async (invalidMongoId) => {
+  it.each(invalidMongoIdTestCases)('returns false when given an invalid facilityId', async (invalidMongoId) => {
```

Comment:
all it statements should begin with 'should' as in `('should return false...)`

Reply:
this one needs to be done too

Reply:
Ah this was outside of my changes (actually didn't mean to correct the `test.` to an `it.` here) I'll change it here in this line anyway but I won't touch the rest of this file as part of this PR.